airflow/serialization/serialized_objects.py

```diff
@@ -254,7 +260,12 @@ def _deserialize_timedelta(cls, seconds: int) -> datetime.timedelta:
         return datetime.timedelta(seconds=seconds)
 
     @classmethod
-    def _value_is_hardcoded_default(cls, attrname: str, value: Any) -> bool:
+    def _is_constcutor_param(cls, attrname: str, instance: Any) -> bool:
```

Comment:
```suggestion
    def _is_constructor_param(cls, attrname: str, instance: Any) -> bool:
```

Reply:
Thanks, was working on (what I thought would be a quick fix) this for 11 hours.